content/en/docs/overview.md

```diff
@@ -1,137 +1,60 @@
 ---
-title: "Project overview"
-description: "An overview of the Notary project"
+title: "Overview"
+description: "An overview of the Notary Project"
 type: docs
 weight: 1
 ---
 
 {{% alert title="Important" color="info" %}}
-This article as well as the rest of this documentation describes the latest status of Notary Project and refers to it as The Notary Project. For documentation of older versions and implementations, please refer to the [previous Notary documentation](https://github.com/notaryproject/notary/tree/master/docs).
+This article as well as the rest of this documentation describes the overview and latest status of the Notary Project. For documentation of older versions and implementations, please refer to the [previous Notary documentation](https://github.com/notaryproject/notary/tree/master/docs).
```

Comment:
I think we may not need to mention [previous Notary](https://github.com/notaryproject/notary/tree/master/docs) at the beginning of this overview doc. It may distract or confuse new users.